Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -139,6 +143,15 @@ class XMLHttpRequestBase {
     }
   }
 
+  _stringToArrayBuffer(str: string): ArrayBuffer {
+    var buffer = new ArrayBuffer(str.length);
+    var bufferView = new Uint8Array(buffer);
```

Comment:
It's not a feature that we need internally, so it's a low priority for us. If you'd like to submit a PR that will degrade gracefully on iOS7 then we'll consider accepting it. Otherwise it will have to wait until we have dropped iOS 7 support from all of our own apps (6% of FB users is still a lot of people).